src/main/java/org/cryptomator/common/settings/Settings.java

```diff
@@ -67,7 +69,9 @@ public class Settings {
 	public final IntegerProperty windowHeight;
 	public final StringProperty language;
 	public final StringProperty mountService;
-	public final StringProperty lastUpdateCheck;
+	public final ObjectProperty<Instant> lastUpdateReminder;
+	public final ObjectProperty<Instant> lastSuccessfulUpdateCheck;
+	public final StringProperty latestVersion;
```

Comment:
I don't believe we should persist `latestVersion`. It should always stay a volatile information, only available when freshly retrieved from our server.